config/jobs/kubernetes-sigs/node-feature-discovery-operator/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+  - ArangoGutierrez
+  - marquiz
+  - zvonkok
+approvers:
+  - ArangoGutierrez
+  - marquiz
+  - zvonkok
```

Comment:
you need to drop non-members from here

Reply:
done, is just you for now, I think once this PR is on, I will apply to be a member and join the OWNERS file